java/piranha/src/test/resources/config/properties_no_method_chain.json

```diff
@@ -0,0 +1,30 @@
+{
+  "methodProperties": [
+    {
+      "methodName": "[flagMethodName].getValue",
```

Comment:
I thought this will be still `[flagName]` but the other one under `flag_method_name` will be `[flagMethodName]`. These seem to be two different entities, but fine if it is being handled in the next PR.